GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -43,15 +45,37 @@ public void CreateFileWithoutClose()
         {
             string srcPath = @"CreateFileWithoutClose.md";
             this.CreateFileWithoutClose(srcPath);
-            this.ValidateGitCommand("status");
+            this.Enlistment.WaitForBackgroundOperations();
+            GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.FileSystem, srcPath);
+            try
+            {
+                this.ValidateGitCommand("status");
+            }
+            catch (Exception ex)
+            {
+                Thread.Sleep(1000);

```

Comment:
Why 1 second?

Reply:
If it's a race condition (hopefully) a second will catch it.  Events are processed in milliseconds.

I don't think there is a race condition, but the new code should help prove it.